ocfweb/account/register.py

```diff
@@ -61,7 +66,7 @@ def request_account(request):
     real_name = directory.name_by_calnet_uid(calnet_uid)
 
     if request.method == 'POST':
-        form = ApproveForm(request.POST)
+        form: Any = ApproveForm(request.POST)
```

Comment:
Why do you need to annoate this as `Any`?

Reply:
Looking into this more, I actually found a preexisting issue.

The original reason for putting `Any` was because mypy complained that ApproveForm doesn't have an attribute `_errors`.

Turns out that assigning to the `_errors` dict directly is deprecated and `add_error()` should be used instead. 

https://stackoverflow.com/questions/188886/inject-errors-into-already-validated-form

Reply:
Awesome!